app/models/course_data/articles_courses.rb

```diff
@@ -19,7 +19,7 @@
 #= ArticlesCourses is a join model between Article and Course.
 #= It represents a mainspace Wikipedia article that has been worked on by a
 #= student in a course.
-class ArticlesCourses < ApplicationRecord
+class ArticlesCourses < ApplicationRecord # rubocop:disable Metrics/ClassLength
```

Comment:
Doesnt make sense to make a refactor to fix this because this class has a lot of duplicated code that will be deleted in the future